clang/test/dpct/memory_management_restricted.cu

```diff
@@ -380,13 +380,35 @@ void foobar() {
 
   size_t free_mem, total_mem;
 
-  //CHECK: dpct::get_current_device().get_memory_info(free_mem, total_mem);
+  // CHECK:  /*
+  // CHECK-NEXT:DPCT1106:{{[0-9]+}}: 'cudaMemGetInfo' was migrated with Intel's Extensions for Device Information which may be not supported by all compilers or runtimes. You may need to adjust the code.
+  // CHECK-NEXT:*/
+  // CHECK-NEXT:dpct::get_current_device().get_memory_info(free_mem, total_mem);
   cudaMemGetInfo(&free_mem, &total_mem);
```

Comment:
I think we also need update diagnostic information in APINames.inc

Reply:
Okay